### PR TITLE
Added a basic heading for cover-letter

### PR DESCRIPTION
**Pull request by sammjainn**
Added a new component named 'Cover' for a basic heading in cover-letter which includes the name & designation of the person.
Screenshot (222)
Screenshot (223)

### CODE REVIEW

File: .vs/ProjectSettings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (venkateshpachigulla) [inline .vs/ProjectSettings.json:1]: what is all about these files, .suo file, .sqlite file ?

Comment (sammjainn) [reply .vs/ProjectSettings.json:1]: I haven't made any changes to these. I searched and it seems they are automatically updated files according to user preferences and what files I had opened etc.

Comment (ManojSatishkumar) [reply .vs/ProjectSettings.json:1]: add unnecessary files to .gitignore

Comment (sammjainn) [reply .vs/ProjectSettings.json:1]: Done. Please review and merge.

File: src/modules/cover/index.js [hunk 2]
```diff
@@ -1,15 +1,17 @@
 // Export the component in this file to display in the preview
 
-import metadata from './metadata'
-
+import metadata from './metadata';
+import head from './head';
+import info from './personal info';
```
Comment (venkateshpachigulla) [inline src/modules/cover/index.js:5]: dont provide spaces for file and folder names. rename it.

**Comment by venkateshpachigulla** [comment]
looks like multiple things are there. Add other screenshots also.

**Pull request merged**